### DISCUSSION TITLE
Is paybamm have 2d and 3d battery module?

### DISCUSSION THREAD

**Discussion by chtld** [pybamm-team/PyBaMM#3365] [Q&A]
Is paybamm have 2d and 3d battery module?

**Comment by chtld** [comment]
I have see it has 2d mesh? but the p2d model is actually a one dimension model.

**Reply by brosaplanella** [reply]
The electrochemical models (DFN, SPM...) are 1D. However, the thermal models can be used for a pouch cell to simulate the 3D behaviour (more details [in this notebook](https://docs.pybamm.org/en/latest/source/examples/notebooks/models/pouch-cell-model.html)).
